apis/cache/v1beta1/cloudmemorystore_instance_types.go

```diff
@@ -187,6 +193,13 @@ type CloudMemorystoreInstanceObservation struct {
 	// for a given instance so should be checked before each import/export
 	// operation.
 	PersistenceIAMIdentity string `json:"persistenceIamIdentity,omitempty"`
+
+	// AuthEnabled: Optional. Indicates whether OSS Redis AUTH is enabled
+	// for the instance. If set to "true" AUTH is enabled on the instance.
+	// Default value is "false" meaning AUTH is disabled.
+	// +optional
+	// +immutable
+	AuthEnabled *bool `json:"authEnabled,omitempty"`
```

Comment:
We represent this value in the spec, and if the user does not set it, it should be late initialized. It can be removed from the status.

Reply:
Removed accordingly.